hazelcast/src/main/java/com/hazelcast/dataconnection/impl/jdbcproperties/DataConnectionProperties.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.dataconnection.impl.jdbcproperties;
+
+/**
+ * The constants that can be used in HZ connections
+ * These need to be translated to a Connection Pool implementation
+ */
+public final class DataConnectionProperties {
+
+    public static final String JDBC_URL = "jdbcUrl";
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
```

Comment:
This should be `user`.

Reply:
Changed to user
rev : e8c93a00e988cc58c8de3934bfdcbc522ce6f04e